ftp/src/test/java/org/apache/ftpserver/filesystem/jimfs/impl/JimfsFtpFile.java

```diff
@@ -236,7 +236,11 @@ public boolean mkdir() {
         boolean retVal = false;
         try {
             if (isWritable()) {
-                Files.createDirectory(path);
+                Set<PosixFilePermission> perms =
+                        PosixFilePermissions.fromString("rwxrwxrwx");
+                FileAttribute<Set<PosixFilePermission>> attr =
+                        PosixFilePermissions.asFileAttribute(perms);
+                Files.createDirectory(path, attr);
```

Comment:
This bit is actually only used in tests within a Unix-configured Jimfs instance. So my take is that it should be safe to do..